packages/patterns-react/src/patterns/PictogramArray/README.md

```diff
@@ -73,6 +73,24 @@ ReactDOM.render(<App />, document.querySelector('#app'));
 > 💡 Don't forget to import the leadspace styles from
 > [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
 
+#### Markdown
+
+Italic is accepted in the copy of the content item using markdown sytanx: single
+`_` (underscore) or `*` (asterisk)
+
+Example:
+
+```javascript
+{
+  copy: '_Lorem_ ipsum *dolor* sit amet.',
+  ...
+}
+```
+
+Renders:
+
+_Lorem_ ipsum _dolor_ sit amet.
+
```

Comment:
For some reason Storybook will add bolder font-weight to their <em> tag.
Apparently others have this issue: https://github.com/tuchk4/storybook-readme/issues/156
I know that there's a story to update storybook, maybe that solves this issue.
Otherwise I can follow up with that open issue in storybook github...